src/modules/map.js

```diff
@@ -38,6 +44,14 @@ class Map {
             case "w":
                 this.spawn(new Wall(id, "wall", column * 20, row * 20, WALL_SIZE, WALL_COLOR));
                 //console.log("Spawned a wall at: (" + row + ", " + column + ")");
+                break;
+			case "sw":
+                this.spawn(new Slow(id, "slowWall", column * 20, row * 20, SLOW_SIZE, SLOW_COLOR));
+                //console.log("Spawned a slow wall at: (" + row + ", " + column + ")");
+                break;
+			case "as":
```

Comment:
We can rename the `as` to `rs` if you choose the name reverse slow. Or keep it here as `as` if you want it to be anti slow.

Similar name comment for consistency as the other ones :)